src/rebar_utils.erl

```diff
@@ -915,7 +915,7 @@ url_append_path(Url, ExtraPath) ->
      case http_uri:parse(Url) of
          {ok, {Scheme, UserInfo, Host, Port, Path, Query}} ->
              {ok, lists:append([atom_to_list(Scheme), "://", UserInfo, Host, ":", integer_to_list(Port),
-                                filename:join(Path, ExtraPath), "?", Query])};
+                                filename:join(Path, ExtraPath), Query])};
```

Comment:
it is retrospectively weird to use `filename:join` here since that might give URLs with backslashes on windows? We can probably keep it but it looks a bit funky

Reply:
FWIW `join` always uses `/` and it does rewrite `\` to `/` on Windows (which should be fine because `\` in URLs should be escaped).